experiments/LES/surfacebubble.jl

```diff
@@ -0,0 +1,206 @@
+using Distributions
+using Random
+using StaticArrays
+using Test
+using DocStringExtensions
+using LinearAlgebra
+
+using CLIMA
+using CLIMA.Atmos
+using CLIMA.GenericCallbacks
+using CLIMA.DGmethods.NumericalFluxes
+using CLIMA.LowStorageRungeKuttaMethod
+using CLIMA.Mesh.Filters
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+using CLIMA.VariableTemplates
+
+import CLIMA.DGmethods: boundary_state!
+import CLIMA.Atmos: atmos_boundary_state!, atmos_boundary_flux_diffusive!, flux_diffusive! 
+import CLIMA.DGmethods.NumericalFluxes: boundary_flux_diffusive!
+
+# -------------------- Surface Driven Bubble ----------------- #
+# Rising thermal driven by a prescribed surface heat flux. 
+# 1) Boundary Conditions: 
+#       Laterally periodic with no flow penetration through top
+#       and bottom wall boundaries. 
+#       Momentum: No flow penetration [NoFluxBC()]
+#       Energy:   Spatially varying non-zero heat flux up to time t₁
+# 2) Domain: 1250m × 1250m × 1000m
+# Configuration defaults are in `src/Driver/Configurations.jl`
+  
+"""
+  SurfaceDrivenBubbleBC <: BoundaryCondition
+Y ≡ state vars
+Σ ≡ diffusive vars
+A ≡ auxiliary vars
+X⁺ and X⁻ refer to exterior, interior faces
+X₁ refers to the first interior node 
+
+# Fields
+$(DocStringExtensions.FIELDS)
+"""
+struct SurfaceDrivenBubbleBC{FT} <: BoundaryCondition
+  "Prescribed MSEF Magnitude [W/m^2]"
```

Comment:
```suggestion
  "Prescribed MSEF Magnitude `[W/m^2]`"
```

Otherwise the doc strings break (due to looking for a link)

Reply:
Thanks - will sort in other instances where this occurs as well.